mobile/src/libs/helpers/get-formatted-time.helper.ts

```diff
@@ -0,0 +1,22 @@
+const SECONDS_IN_MINUTE = 60;
+const TARGET_STRING_LENGTH = 2;
+
+const getFormattedTime = (time: number): string => {
+  if (time && !Number.isNaN(time)) {
+    const minutes = Math.floor(time / SECONDS_IN_MINUTE);
+    const formattedMinutes = minutes
+      .toString()
+      .padStart(TARGET_STRING_LENGTH, '0');
```

Comment:
Please check implementation from frontend side, your handlers are good but we will have one style for best reusing code)

Reply:
As @OlenaYatseniuk already mentined)

Reply:
Then I guess I'll wait until this task on the frontend will be merged

Reply:
Merged